landingai/visualize.py

```diff
@@ -15,7 +15,7 @@
 
 
 def overlay_predictions(
-    predictions: list[Prediction],
+    predictions: List[Prediction],
```

Comment:
nit: `List` is a legacy feature for old python, it's deprecated in python 3.9. so we actually prefer list over List.
see: https://docs.python.org/3/library/typing.html#typing.List

Reply:
Thanks!  Fixed